### PR TITLE
Disable AuthorizeSauce's debug setting when in production [PD-3235]

### PR DESCRIPTION
**Pull request by JLMcLaugh**
## What's this PR do?
Disables the debug setting for credit card authorization when in production. Credit card transactions are being rejected because we're posting to authorize.net's sandbox api.
## How should this be manually tested?
The best we can do for now in staging is verify we haven't broken the sandbox api requests. We can create a payed product for apply.directemployers.org in the staging environment and test that we're able to purchase it from another company.

### CODE REVIEW

File: postajob/payment.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from authorize.client import AuthorizeClient, AuthorizeCreditCard
 from authorize.data import Address, CreditCard
 
+from django.conf import settings
 
 def get_client(authorize_id, authorize_key):
-    return AuthorizeClient(authorize_id, authorize_key)
+    # Use the sandbox api if we're not in production
+    debug = (ENVIRONMENT != "Production")
```
Comment (tdphillips) [inline postajob/payment.py]: `settings.ENVIRONMENT`?

Comment (JLMcLaugh) [inline postajob/payment.py]: Ugh, thanks.

File: postajob/payment.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 from authorize.client import AuthorizeClient, AuthorizeCreditCard
 from authorize.data import Address, CreditCard
 
+from django.conf import settings
 
 def get_client(authorize_id, authorize_key):
-    return AuthorizeClient(authorize_id, authorize_key)
+    # Use the sandbox api if we're not in production
+    debug = (settings.ENVIRONMENT != "Production")
```
Comment (darrint) [inline postajob/payment.py]: Having a single setting that has (potentially) more than one effect can be a problem. Can we make a setting specifically for this?

Comment (tdphillips) [inline postajob/payment.py]: Why would it be a problem?

Comment (JLMcLaugh) [inline postajob/payment.py]: That makes one vote from @almathew for using DEBUG and one vote for using something more specific. I'm inclined to call this a happy medium unless someone really objects.

**Pull request merged**